### DISCUSSION TITLE
How to resolve this error

### DISCUSSION THREAD

**Discussion by raviteja-salva198** [community/community#141573] [Other Feature Feedback, Questions, & Ideas]
Enumerating objects: 23, done.
Counting objects: 100% (23/23), done.
Compressing objects: 100% (20/20), done.
Writing objects: 100% (22/22), 152.65 KiB | 3.32 MiB/s, done.
Total 22 (delta 3), reused 0 (delta 0), pack-reused 0 (from 0)
error: RPC failed; HTTP 400 curl 22 The requested URL returned error: 400
send-pack: unexpected disconnect while reading sideband packet
fatal: the remote end hung up unexpectedly
Everything up-to-date

**Comment by tmduy** [comment]
The specific error RPC failed; HTTP 400 suggests that the server rejected the request, often due to network or repository configuration issues.

Solution: Ask a repository owner to change the config.

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @raviteja-salva198 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
